sdks/go/pkg/beam/io/bigqueryio/bigquery.go

```diff
@@ -198,33 +228,46 @@ func (f *writeFn) ProcessElement(ctx context.Context, _ int, iter func(*beam.X)
 		return err
 	}
 
+	schema := mustInferSchema(f.Type.T)
 	table := dataset.Table(f.Table.Table)
 	if _, err := table.Metadata(ctx); err != nil {
 		if !isNotFound(err) {
 			return err
 		}
-		if err := table.Create(ctx, &bigquery.TableMetadata{Schema: mustInferSchema(f.Type.T)}); err != nil {
+		if err := table.Create(ctx, &bigquery.TableMetadata{Schema: schema}); err != nil {
 			return err
 		}
 	}
 
 	var data []reflect.Value
+	// This stores the running byte size estimate of a BQ request. Initializing to 1KB
+	// as an estimate for overall message overhead.
+	var size = 1024
```

Comment:
Please move the 1024 to a constant since it's being used in two places.

Reply:
Done.